keyboards/manyboard/macro/keymaps/default/keymap.c

```diff
@@ -0,0 +1,73 @@
+/* Copyright 2020 William Ehman
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT_ortho_4x4( 
+    KC_F1,  KC_F2,  KC_F3,  KC_F4, 
+    KC_F5,  KC_F6,  KC_F7,  KC_F8, 
+    KC_F9,  KC_F10, KC_F11, KC_F12, 
+    KC_F13, KC_F14, KC_F15, TO(1)
+  ),
+  [1] = LAYOUT_ortho_4x4( 
+    KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS, 
+    KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS, 
+    KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS, 
+    KC_TRNS,  KC_TRNS,  KC_TRNS,  TO(2)
+  ),
+  [2] = LAYOUT_ortho_4x4( 
+    KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS, 
+    KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS, 
+    KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS, 
+    KC_TRNS,  KC_TRNS,  KC_TRNS,  TO(0)
+  ),
+};
+
+layer_state_t layer_state_set_user(layer_state_t state) {
```

Comment:
I would recommend the use of the "lighting layers" feature over this, as this is more intensive. 
https://docs.qmk.fm/#/feature_rgblight?id=lighting-layers

Reply:
I haven't been able to get lighting layers to work properly, is it alright to leave it as is since it's only changing on 1 LED between 3 layers as is?